.github/workflows/publish.yml

```diff
@@ -11,34 +11,73 @@ on:
     paths:
       - "src/**"
       - ".github/workflows/publish.yml"
+  repository_dispatch:
+    types: [create-release]
+    client_payload:
+      release_tag:
+        required: true
 
 jobs:
+  create_release:
+    if: ${{ github.event_name == 'repository_dispatch' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: Brightspace/third-party-actions@actions/checkout
+      - name: Create release
+        env:
+          RELEASE_TAG: ${{ github.event.client_payload.release_tag }}
+        run: |
+          gh release create "$RELEASE_TAG" --draft `
```

Comment:
`--draft` because I accidentally broke someone's workflow process by creating non-drafted releases 😓

Reply:
Yea for the jumpboxes they use the latest version release apparently and me creating releases while testing was making them pull the dotnet version

Reply:
yeah this is not surprising. one should not change the "latest" release of a GitHub repo lightly